src/intrprtr.c

```diff
@@ -2697,7 +2697,7 @@ void            IntrAssDVar (
 
     if ( STATE(IntrCoding) > 0 ) {
         ErrorQuit( "Variable: <debug-variable-%d-%d> cannot be used here",
-                   dvar >> 10, dvar & 0x3FF );
+                   dvar >> MAX_FUNC_LVARS_BITS, dvar & MAX_FUNC_LVARS_MASK );
```

Comment:
Note that we replace 10 by 16 here (also in the mask). But that's actually correct: The limit used to be 1024 at some point, and was raised to 65536 later, but these four ErrorQuit calls were not adapted.